components/ide/jetbrains/backend-plugin/build.gradle.kts

```diff
@@ -117,6 +117,7 @@ tasks {
     }
 
     runPluginVerifier {
+        enabled = false
```

Comment:
Temporarily disabled due to [this error](https://werft.gitpod-dev.com/job/gitpod-build-jetbrains-update-stable-ides.28/raw):
```
Cannot download 'IC-2022.3' from 'release' channel: https://cache-redirector.jetbrains.com/download.jetbrains.com/idea/ideaIC-2022.3.tar.gz.
```

Reply:
Re-enabled in:
- https://github.com/gitpod-io/gitpod/pull/15273